nat-lab/tests/helpers.py

```diff
@@ -286,11 +287,13 @@ async def setup_environment(
 
     try:
         yield Environment(api, nodes, connection_managers, clients)
-    finally:
+
+        print(datetime.now(), "Checking connection limits")
```

Comment:
Is this print needed here ?

Reply:
IMHO, it makes it explicit when the connections limits are being checked in tests logs. Which helps a bit when analyzing failures.